monai/utils/module.py

```diff
@@ -210,12 +210,14 @@ def instantiate(path: str, **kwargs):
     """
 
     component = locate(path)
+    if component is None:
+        raise ModuleNotFoundError(f"Cannot locate '{path}'.")
```

Comment:
is it ok to raise an error here? @Nic-Ma

Reply:
Looks good to me as we already covered all the MONAI classes and functions in the locate unit test.

Thanks.